app/controllers/provider_interface/provider_interface_controller.rb

```diff
@@ -97,17 +97,26 @@ def redirect_if_setup_required
       if @provider_setup.next_agreement_pending
         redirect_to provider_interface_new_data_sharing_agreement_path
       elsif @provider_setup.next_relationship_pending
-        redirect_to provider_interface_provider_relationship_permissions_organisations_path
+        redirect_to setup_organisation_permissions_path
       end
     end
 
     def performing_provider_organisation_setup?
       [
         ProviderInterface::ProviderAgreementsController,
         ProviderInterface::ProviderRelationshipPermissionsSetupController,
+        ProviderInterface::OrganisationPermissionsSetupController,
```

Comment:
If we use the feature flag here to decide which controller to use, then we don't need to redirect away in the individual controllers. Maybe that's better